### PR TITLE
Image Modeling Example Scenario

### PR DESCRIPTION
**Pull request by adamboeglin**

### CODE REVIEW

File: docs/example-scenario/infrastructure/image-modeling.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+titleSuffix: Azure Example Scenarios
+description: Accelerate digital image-based modeling on Azure using Avere and Agisoft PhotoScan
+author: adamboeglin
+ms.date: 1/11/2019
+ms.custom: cat-team
+---
+
+# Accelerate digital image-based modeling on Azure
+
+This example scenario provides architecture and design guidance for any organization that wants to perform image-based modeling on Azure infrastructure as a service (IaaS). The scenario is designed for running photogrammetry software on Azure Virtual Machines (VMs) using high-performance storage that accelerates processing time. The environment can be scaled up and down as needed and supports terabytes of storage without sacrificing performance.
```
Comment (Petertaylor9999) [inline docs/example-scenario/infrastructure/image-modeling.md]: While it's not in the style guide, convention is for "infrastructure as a service" to be "infrastructure-as-a-service".

File: docs/example-scenario/infrastructure/image-modeling.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+### Components
+
+- [Agisoft PhotoScan](http://www.agisoft.com/): The PhotoScan Scheduler runs on a Windows 2016 Server VM, and the processing nodes use five VMs with GPUs that run CentOS Linux 7.5.
+- [Avere vFXT](/azure/avere-vfxt/avere-vfxt-overview) is a file caching solution that uses object storage and traditional network-attached storage (NAS) to optimize storage of large datasets. It includes:
+  - Avere Controller. This VM executes the script that installs the Avere vFXT cluster and runs Ubuntu 18.04 LTS. The VM can be used later to add or remove cluster nodes and to destroy the cluster as well.
+  - vFXT cluster. At least three VMs are used, one for each of the Avere vFXT nodes based on Avere OS 5.0.2.1. These VMs form the vFXT cluster, which is attached to Azure Blob storage.
+- [Active Directory domain controllers](/azure/active-directory-domain-services/active-directory-ds-overview) allow the host access to domain resources and provide DNS name resolution. Avere vFXT adds _x_ number of A records—for example, each A record in a vFXT cluster points to the IP address of each Avere vFXT node. In this setup, all VMs use the round-robin pattern to access vFXT exports.
```
Comment (Petertaylor9999) [inline docs/example-scenario/infrastructure/image-modeling.md]: It looks like these are VMs running Active Directory, but the link is to Azure AD Domain Services (which is a service). Can either one be used here?

File: docs/example-scenario/infrastructure/image-modeling.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+- [Other VMs](/azure/virtual-machines/) serve as jump boxes used by the administrator to access the scheduler and processing nodes. The Windows jumpbox is mandatory to allow the administrator to access the head node via remote desktop protocol. The second jumpbox is optional and runs Linux for administration of the worker nodes.
+- [Network security groups](/azure/virtual-network/manage-network-security-group) (NSGs) on the Jumpbox subnet allow ports 3389 and 22 access to the VMs attached to the Jumpbox subnet.
```
Comment (Petertaylor9999) [inline docs/example-scenario/infrastructure/image-modeling.md]: May want to mention that the NSG allows access to the PIP only.

File: docs/example-scenario/infrastructure/image-modeling.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+Consider the following options to further improve security in this scenario:
+
+- Use network virtual appliances such as Fortinet, Checkpoint, and Juniper.
+- Apply [role-based access control](/azure/role-based-access-control/overview) to the resource groups.
+- Enable VM [JIT](/azure/security-center/security-center-just-in-time) access if jump boxes are accessed via the Internet.
+- Use [Azure Key Vault](/azure/key-vault/quick-create-portal) to store the passwords used by administrator accounts.
+
+## Pricing
+
+The cost of running this scenario can vary greatly depending on on multiple factors.  Choices such as using [GPU-optimized](/azure/virtual-machines/linux/sizes-gpu) VMs (like this example), how much storage is required, and the amount of time to complete a job will determine your cost.
```
Comment (Petertaylor9999) [inline docs/example-scenario/infrastructure/image-modeling.md]: The second sentence is a bit confusing. It's enumerating things that must be chosen, but the second clause is a requirement. I'd recast it to make clearer that the choice of storage type or allocation affects cost.

File: docs/example-scenario/infrastructure/image-modeling.md (hunk 1 above)
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: fixed.

File: docs/example-scenario/infrastructure/image-modeling.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+# Accelerate digital image-based modeling on Azure
+
+This example scenario provides architecture and design guidance for any organization that wants to perform image-based modeling on Azure infrastructure as a service (IaaS). The scenario is designed for running photogrammetry software on Azure Virtual Machines (VMs) using high-performance storage that accelerates processing time. The environment can be scaled up and down as needed and supports terabytes of storage without sacrificing performance.
+
+## Relevant use cases
+
+Relevant use cases include:
+
+- Modeling and measuring buildings, engineering structures, and forensic and accident scenes.
```
[the remark this answers is not in the document]
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: good catch, probably doesn't need the and.

File: docs/example-scenario/infrastructure/image-modeling.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+- Creating visual effects for computer games and movies.
+- Using digital images to generate indirect measurements of objects of various scales as in urban planning and other applications.
```
[the remark this answers is not in the document]
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: Agreed.

File: docs/example-scenario/infrastructure/image-modeling.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+## Architecture
+
+This example describes the use of Agisoft PhotoScan photogrammetry software backed by Avere vFXT storage. PhotoScan was chosen for its popularity in geographic information system (GIS) applications, cultural heritage documentation, game development, and visual effects production. It is suitable for both close-range photogrammetry and aerial photogrammetry.
+
+The concepts in this article apply to any high-performance computing (HPC) workload based on a scheduler and worker nodes managed as infrastructure.  For this workload, Avere vFXT was selected for its superior performance during benchmark tests.  However, the scenario decouples the storage from the processing so that other storage solutions can be used (see [alternatives](#alternatives) later in this document).
+
+This architecture also includes Active Directory domain controllers to control access to Azure resources and internal name resolution through the Domain Name System (DNS). Jump boxes provide administrator access to the Windows and Linux VMs that run the solution.
```
[the remark this answers is not in the document]
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: Fixed.

File: docs/example-scenario/infrastructure/image-modeling.md (hunk 2 above)
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: Need to follow up with the SME, but I believe either could be used.

File: docs/example-scenario/infrastructure/image-modeling.md (hunk 4 above)
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: Agreed, cleaned it up slightly to provide clarity.

File: docs/example-scenario/infrastructure/image-modeling.md (hunk 2 above)
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: Should not be azure AD, fixed link.

File: docs/example-scenario/infrastructure/image-modeling.md (hunk 3 above)
Comment (adamboeglin) [reply docs/example-scenario/infrastructure/image-modeling.md]: Done.

**Comment by adamboeglin** [comment]
@Petertaylor9999, comments have been resolved.

**Pull request merged**